x-pack/plugins/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -22,7 +22,7 @@ export interface QueryEventsBySavedObjectResult {
   page: number;
   per_page: number;
   total: number;
-  data: IEvent[];
+  data: IValidatedEvent[];
```

Comment:
`IValidatedEvent` is a little nicer to use as a result of the query since it's not partial-ized or writable.